truenas/webui
# NAS-111734 / 21.08 / Show resources that depend on the dataset before deleting

How to test?

- Create a new dataset
- Create share using the dataset
- Choose 'Delete Dataset' on the Storage page
- Make sure that share is listed in the confirmation dialog

Example:
![image](https://user-images.githubusercontent.com/351613/128714561-a12f2872-5859-465f-89f5-87f38743e907.png)
